packages/webdriverio/src/utils/refetchElement.js

```diff
@@ -0,0 +1,27 @@
+export default function refetchElement () {
+    let currentElement = this;
+
+    return new Promise ( async () => {
+        let selectors = [];
+
+        //Crawl back to the browser object, and cache all selectors
+        while(currentElement.elementId && currentElement.parent) {
+            selectors.push(currentElement.selector);
+            currentElement = currentElement.parent;
+        }
+        selectors.reverse();
+
+        // Beginning with the browser object, rechain
+        const element = await selectors.reduce(async (elementPromise, selector) => {
+            const resolvedElement = await elementPromise;
+            const newElement = await resolvedElement.$(selector);
+            await newElement.waitForExist();
```

Comment:
`waitForExist` should not be necessary here, if you fetch the element the line before and it is not there, it would throw an error

Reply:
Removed.